Source/MonoGame.Extended/Graphics/PrimitiveTypeExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using Microsoft.Xna.Framework.Graphics;
+
+namespace MonoGame.Extended.Graphics
+{
+    public static class PrimitiveTypeExtensions
+    {
+        public static int GetPrimitiveCount(this PrimitiveType primitiveType, int verticesOrIndicesCount)
+        {
+            switch (primitiveType)
+            {
+                case PrimitiveType.LineStrip:
+                {
```

Comment:
Putting these extra { 's in the case statements takes up a lot of extra lines for no reason. I know it's a personal preference. I'm just not a fan of having extra code that doesn't do anything soley for layout purposes.